packages/radix-ui-themes/src/components/checkbox-card-group.css

```diff
@@ -0,0 +1,168 @@
+@import './base-checkbox.css';
+
+.rt-CheckboxCardGroupRoot {
+  line-height: var(--line-height);
+  letter-spacing: var(--letter-spacing);
+  user-select: none;
+  cursor: default;
+}
+
+.rt-CheckboxCardGroupItem {
+  display: flex;
+  align-items: center;
+  gap: var(--checkbox-card-group-item-gap);
+  box-sizing: border-box;
+  padding: var(--checkbox-card-group-item-padding-y) var(--checkbox-card-group-item-padding-x);
+  padding-right: calc(
+    var(--checkbox-card-group-item-padding-x) + var(--checkbox-card-group-item-checkbox-size) +
+      var(--checkbox-card-group-item-gap)
+  );
```

Comment:
I'd do `calc(var(--checkbox-card-group-item-padding-x) * 2 + var(--checkbox-card-group-item-checkbox-size));` to keep the checkbox looking a bit more natural with equal whitespace on the left and on the right around it. It feels more pinned to the right than part of the flex with the item.

<img width="708" alt="image" src="https://github.com/radix-ui/themes/assets/8441036/963ee117-48b3-4a1e-b637-8b7c8c83af23">

Reply:
Yes, that's better, thanks!